integration_tests/src/main/python/data_gen.py

```diff
@@ -1030,3 +1030,10 @@ def append_unique_to_rows(rows):
     new_rows = append_unique_to_rows(collected)
     new_schema = StructType(existing_schema.fields + [StructField("uniq_int", IntegerType(), False)])
     return spark.createDataFrame(new_rows, new_schema)
+
+# add multiple confs
+def add_conf(*confs):
```

Comment:
Do we really need this?  We already have `copy_and_update` whichcan be used to accomplish the same thing and is used in many other places, e.g.:
```
combined_confs = copy_and_update(first_conf, second_conf, third_conf ...)
```